src/server/game/Spells/SpellInfoCorrections.cpp

```diff
@@ -4466,7 +4466,8 @@ void SpellMgr::LoadSpellInfoCorrections()
     });
 
     // Revitalize
-    ApplySpellFix({ 48540, 48541, 48542, 48543 }, [](SpellInfo* spellInfo)
+    // Ignite
+    ApplySpellFix({ 48540, 48541, 48542, 48543, 12654 }, [](SpellInfo* spellInfo)
```

Comment:
I'm assuming that SPELL_ATTR1_NO_THREAT removes all threat caused.
If so then it means this is incorrect. Ignite does cause threat.